clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -11084,9 +11084,7 @@ void LinkerWrapper::ConstructJob(Compilation &C, const JobAction &JA,
     for (auto &I :
          llvm::make_range(ToolChainRange.first, ToolChainRange.second)) {
       const ToolChain *TC = I.second;
-      // TODO: Third party AOT support needs to be added in new offloading
-      // model.
-      if (TC->getTriple().isSPIROrSPIRV()) {
+      if (TC->getTriple().isSPIROrSPIRV() || TC->getTriple().isNVPTX()) {
```

Comment:
For AMD, we do not pass any SYCL device libraries.

Reply:
This comment seems useful for code readers not just PR reviewers.

Reply:
- Ah..you are right. Let me take a look at all these comments and move them into the code if it seems like a better fit.
- Thanks so much for looking closer.

Sincerely